x-pack/plugins/cloud_security_posture/server/create_indices/create_indices.ts

```diff
@@ -11,19 +11,20 @@ import {
   BENCHMARK_SCORE_INDEX_PATTERN,
   BENCHMARK_SCORE_INDEX_TEMPLATE_NAME,
   CLOUD_SECURITY_POSTURE_PACKAGE_NAME,
-  CSP_INGEST_TIMESTAMP_PIPELINE,
   FINDINGS_INDEX_NAME,
   LATEST_FINDINGS_INDEX_DEFAULT_NS,
   LATEST_FINDINGS_INDEX_PATTERN,
   LATEST_FINDINGS_INDEX_TEMPLATE_NAME,
 } from '../../common/constants';
 import { createPipelineIfNotExists } from './create_processor';
 import { benchmarkScoreMapping } from './benchmark_score_mapping';
+import { latestFindingsPipelineIngestConfig, scorePipelineIngestConfig } from './ingest_pipelines';
 
 // TODO: Add integration tests
 
 export const initializeCspIndices = async (esClient: ElasticsearchClient, logger: Logger) => {
-  await createPipelineIfNotExists(esClient, CSP_INGEST_TIMESTAMP_PIPELINE, logger);
+  await createPipelineIfNotExists(esClient, scorePipelineIngestConfig, logger);
+  await createPipelineIfNotExists(esClient, latestFindingsPipelineIngestConfig, logger);
```

Comment:
i _think_ we can wrap these two in a `Promise.all` too? unless they do depend on each other